### PR TITLE
Cheat sheet

### PR DESCRIPTION
**Pull request by eduard93**
For https://github.com/intersystems-ru/cos-guidelines/issues/16

### CODE REVIEW

**Comment by eduard93** [comment]
See cheat [here](https://github.com/intersystems-ru/cos-guidelines/blob/cheat-sheet/CHEAT.md)

**Comment by adaptun** [comment]
Is this learning services cheat sheet? Are they OK with putting it online?

File: CHEAT.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+| Assert condition is true    | Do $$$AssertTrue(condition, message)           |
+| Assert condition isn't true | Do $$$AssertNotTrue(condition, message)        |
+| Log message                 | Do $$$LogMessage(message)                      |
+
+## Other Macros
+
+| Action                           | Code                                    |
+|----------------------------------|-----------------------------------------|
+| Return a good status             | Quit $$$OK                              |
+| Return an error status           | Quit $$$ERROR($$$GeneralError, message) |
```
Comment (daimor) [inline CHEAT.md]: $$$ThrowOnError(status)

**Comment by eduard93** [comment]
@adaptun It's based on foundations course, but source file does not have `confidential` written anywhere on it. @evshvarov any ideas on how to proceed?

File: CHEAT.md (hunk 1 above)
Comment (eduard93) [inline CHEAT.md]: And $$$TOE too.

Comment (eduard93) [inline CHEAT.md]: Added in https://github.com/intersystems-ru/cos-guidelines/commit/7e42b52bd7190b187a0aebbda97f7c27f33736ca

**Pull request merged**